Foundation/URLSession/http/HTTPURLProtocol.swift

```diff
@@ -365,392 +360,9 @@ fileprivate extension _HTTPURLProtocol._Body {
-        }
-    }
-
-    func fill(writeBuffer buffer: UnsafeMutableBufferPointer<Int8>) -> _EasyHandle._WriteBufferResult {
-        guard case .transferInProgress(let ts) = internalState else { fatalError("Requested to fill write buffer, but transfer isn't in progress.") }
-        guard let source = ts.requestBodySource else { fatalError("Requested to fill write buffer, but transfer state has no body source.") }
-        switch source.getNextChunk(withLength: buffer.count) {
-        case .data(let data):
-            copyDispatchData(data, infoBuffer: buffer)
-            let count = data.count
-            assert(count > 0)
-            notifyDelegate(aboutUploadedData: Int64(count))
-            return .bytes(count)
-        case .done:
-            return .bytes(0)
-        case .retryLater:
-            // At this point we'll try to pause the easy handle. The body source
-            // is responsible for un-pausing the handle once data becomes
-            // available.
-            return .pause
-        case .error:
-            return .abort
-        }
-    }
-
-    func transferCompleted(withErrorCode errorCode: Int?) {
-        // At this point the transfer is complete and we can decide what to do.
-        // If everything went well, we will simply forward the resulting data
-        // to the delegate. But in case of redirects etc. we might send another
-        // request.
-        guard case .transferInProgress(let ts) = internalState else { fatalError("Transfer completed, but it wasn't in progress.") }
-        guard let request = task?.currentRequest else { fatalError("Transfer completed, but there's no current request.") }
-        guard errorCode == nil else {
-            internalState = .transferFailed
-            failWith(errorCode: errorCode!, request: request)
-            return
-        }
-
-        if let response = task?.response as? HTTPURLResponse {
-            var transferState = ts
-            transferState.response = response
-        }
-
-        guard let response = ts.response else { fatalError("Transfer completed, but there's no response.") }
-        internalState = .transferCompleted(response: response, bodyDataDrain: ts.bodyDataDrain)
-        let action = completionAction(forCompletedRequest: request, response: response)
-        
-        switch action {
-        case .completeTask:
-            completeTask()
-        case .failWithError(let errorCode):
-            internalState = .transferFailed
-            failWith(errorCode: errorCode, request: request)
-        case .redirectWithRequest(let newRequest):
-            redirectFor(request: newRequest)
-        }
-    }
-
-    func seekInputStream(to position: UInt64) throws {
-        // We will reset the body sourse and seek forward.
-        NSUnimplemented()
-    }
- 
-    func updateProgressMeter(with propgress: _EasyHandle._Progress) {
-        //TODO: Update progress. Note that a single URLSessionTask might
```

Comment:
Likewise this comment is missing in the moved code. Please retain existing comments.

Reply:
Same as above